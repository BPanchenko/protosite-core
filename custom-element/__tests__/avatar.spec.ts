import AvatarElement from '../avatar';

describe('Avatar', () => {
	const avatar = new AvatarElement();
	test('renders correctly', () => {
		expect(avatar).toMatchSnapshot();
	});
});
