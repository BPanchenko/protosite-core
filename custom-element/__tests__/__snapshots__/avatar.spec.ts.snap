// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders correctly 1`] = `
<c-avatar
  class="u-display-contents"
/>
`;
